src/graphics/device.js

```diff
@@ -2092,6 +2092,10 @@ Object.assign(pc, function () {
                             this.instancedAttribs[locationId] = false;
                         }
                     }
+                    else
+                    {
+                        gl.disableVertexAttribArray(attribute.locationId); //this allows for a single variant for NodeMaterials
+                    }
```

Comment:
there's different (similar) line like this in master (used by morphing), hopefully that would work for you

Reply:
Thanks - just merged master and will test with your version.

Reply:
It works